components/LandingPage.tsx

```
@@ -0,0 +1,331 @@
+              Real Flow is your
+              <br />
+              <span className="text-neutral-800">personal task manager</span>
+            </h1>
+            
+            <p className="text-xl text-neutral-800 mb-8 max-w-2xl mx-auto leading-relaxed">
+              Stay on top of everyday tasks. Streamline your workflow and 
+              boost productivity with our intuitive task management platform.
+            </p>
+            
+            <div className="flex flex-col sm:flex-row gap-4 justify-center items-center">
+              <button className="bg-neutral-800 text-white px-8 py-4 rounded-lg font-semibold hover:bg-neutral-700 transition-all transform hover:scale-105 flex items-center gap-2">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)